sdk/trace/trace_test.go

```diff
@@ -393,7 +393,7 @@ func TestSetSpanAttributesOnStart(t *testing.T) {
 	span := startSpan(tp,
 		"StartSpanAttribute",
 		trace.WithAttributes(attribute.String("key1", "value1")),
-		trace.WithAttributes(attribute.String("key2", "value2")),
+		trace.WithAttributes(attribute.String("key1", "value2")),
```

Comment:
Why did you change this test?

Reply:
Because there are two attributes set originally, the order of those attributes is a something checked indirectly. That order is not guaranteed anymore.

This updates the test to still set two attributes, similar to the original, but it updates the key-value. Resulting in a single attribute for the exported span. This update behavior is something guaranteed and is functionality also required by the specification.

Reply:
That makes me think that our assert isn't comprehensive enough for our test cases.

But, we cover this behavior in another test, so I won't hold this up.